config/CommandDispatcherImplCfg.hpp

```diff
@@ -13,6 +13,7 @@
 enum {
     CMD_DISPATCHER_DISPATCH_TABLE_SIZE = 100, // !< The size of the table holding opcodes to dispatch
     CMD_DISPATCHER_SEQUENCER_TABLE_SIZE = 25, // !< The size of the table holding commands in progress
+    ALLOW_REREGISTRATION = false // !< If true will reregister opcodes otherwise will ASSERT for duplicate opcode
```

Comment:
Do we need to make re-registering the opcode/port combination configurable? I don't think there's any situations where this behavior could be dangerous or undesirable, so we might be able to get away with just making this the new behavior and avoid adding another configuration option.

Reply:
@LeStarch, is it OK to remove the Cfg option?

Reply:
@saba-ja yes, in the case where we check specifically the same opcode and port.  The only danger is wasted cycles if users call register too many times....but the behavior will be correct.